MeshLib/Node.h

```diff
@@ -67,6 +67,9 @@ class Node final : public MathLib::Point3dWithID
         return Node(_x[0]-v[0], _x[1]-v[1], _x[2]-v[2]);
     }
 
+    /// Sets the ID of a node to the given value.
+    void setID(std::size_t id) { _id = id; }
```

Comment:
As discussed once before, moving `setID` into the public interface considerably softens the invariants of the class. (There are structures relying on the node's id).
Is it possible to keep `setID` in the protected part of the interface?